pkg/metrics/kubelet_metrics.go

```diff
@@ -96,6 +96,30 @@ var KnownKubeletMetrics = map[string][]string{
 	"rest_client_request_status_codes":                       {"code", "host", "method"},
 }
 
+var InterestingKubeletMetricsForE2E = []string{
```

Comment:
Same, your putting an E2E symbol into the main libs, just trying to understand the logic of the shuffle.